scraper/data/errors.csv

```diff
@@ -0,0 +1,4790 @@
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+"Not Found",,,,
+,-2,"open","ENOENT","/Users/aidnurs/coding/bachelor/archieve-parser/archive.org-recovery/wiki/Biogenic_Decalcifier/index.html"
```

Comment:
Why is this error differently to other "Not Found" errors? Doesn't sound like a plant.

Reply:
These are 404 pages, while the "Not Found" errors indicate listing pages as I described in the comment above